### ISSUE TITLE
WordPress Installation Cleanup

### ISSUE THREAD

**Issue by magicspacekiwi** [creativedisturbance/podcasts#658]
Even on a blazingly fast server, CD is still unacceptably slow.

This is issue is for discussion and updates on progress in improving performance.

**Comment by magicspacekiwi** [comment]
I've removed several plugins from the site.
No significant performance gains noted, but still probably a good thing to do.

## List of purged plugins

### Previously inactive

- Advanced Code Editor
- Better WordPress Minify
- Better WordPress reCAPTCHA
- Bowe Codes
- BuddyPress
- CodePen Embedded Pens Shortcode
- Download Theme
- Duplicator
- LayerSlider WP
- Post Duplicator
- Really Simple CAPTCHA
- Revolution Slider
- User Access Manager
- WP Theme File Uploader
- WPMU DEV Dashboard

### Previously active, problematic
- WP Super Cache _(removing due to lack of benefit observed considering added complexity)_

### Previously active, unused
- MailChimp

**Comment by magicspacekiwi** [comment]
Previously the MySQL tables for CD were using the inefficient MyISAM engine.
I've switched them all to InnoDB which is more performant as it locks per-row instead of per-table.

Unfortunately, the performance benefit is still tiny compared to the massive ~10 sec TTFB for CD.
Still hunting down the root cause of that.

**Comment by magicspacekiwi** [comment]
Progress!

After observing all queries to the  database with the MySQL general query logging utility, I discovered a pattern - WordPress was pulling some base64 encoded junk from wp_options from an option called wp_data_newa which from what I can tell is a remnant from an old malware infection on the site.

I successfully removed it, and load times have gone from ~ 13 seconds to ~ 3 seconds.
I think we can still do better, though.

@cassininazir

**Comment by cassininazir** [comment]
Excellent progress. The innodb setup in previous post might from the cpanel installer?

Should we also explore the other sites for similar issues?

Sent from my iPhone

On Mar 31, 2017, at 7:18 PM, Colin Campbell <notifications@github.com<mailto:notifications@github.com>> wrote:


Progress!

After observing all queries to the database with the MySQL general query logging utility, I discovered a pattern - WordPress was pulling some base64 encoded junk from wp_options from an option called wp_data_newa which from what I can tell is a remnant from an old malware infection on the site.

I successfully removed it, and load times have gone from ~ 13 seconds to ~ 3 seconds.
I think we can still do better, though.

@cassininazir<https://github.com/cassininazir>

-
